content/academy/apify_platform/deploying/apify_cli.md

```diff
@@ -1,14 +1,22 @@
 ---
 title: Apify CLI
-description: Learn about and install an important tool for communicating with your Apify SDK projects and the Apify Platform from your terminal - the Apify CLI.
-menuWeight: 1
+description: Learn about and install an important tool for communicating with your Apify projects and the Apify platform from your terminal - the Apify CLI.
+menuWeight: 2
 paths:
 - apify-platform/deploying/apify-cli
 ---
 
 # [](#the-cli) Apify CLI
 
-The [Apify CLI](https://docs.apify.com/cli) helps you create, develop, build and run Apify actors, and manage the Apify cloud platform from any computer. It can be used to automatically generate the boilerplate for different types of projects, remotely call actors on the platform, and run your own projects. The Apify CLI will become your best friend while developing locally with the [Apify SDK](https://sdk.apify.com/).
+At this point, our project has three files:
+
+- **index.js**
+- **package.json**
+- **Dockerfile**
+
+There is one other file we need to make it work, and that is the **apify.json** file. We could write this by hand; however, it can be automated by using the Apify CLI, which is a tool we'll be using later on for other purposes. Therefore, it's good to get it installed on our machines now so we don't have to worry about it later.
+
+The [Apify CLI](https://docs.apify.com/cli) helps you create, develop, build and run Apify actors, and manage the Apify cloud platform from any computer. It can be used to automatically generate the boilerplate for different types of projects, initialize projects, remotely call actors on the platform, and run your own projects. The Apify CLI will become your best friend while developing for the Apify platform.
```

Comment:
I am having them all install it at this point in the course regardless of preferred language specifically because of the `apify init` (which is used in the lesson) and `apify push` commands. I'm having them log in so the `APIFY_TOKEN` environment variable is present as well (it is used in the **Inputs & outputs** lesson). These are the reasons I've mushed the Apify CLI lesson and the project initialization lessons into one unified comprehensive lesson.

Reply:
I don't think we will add the ability to `apify run` other than JS code anytime soon.

Also, the `apify.json` file will become obsolete soon, so I wouldn't focus on it too much. Since this is about deploying, I'd focus on `apify push` as that is the command that actually deploys the actor. `apify.json` is just some artifact that nobody really cares about 😄

Reply:
I read the whole thing and finally understand what the fuss about `apify.json` is.

👉 You don't need `apify.json` if you're not using the `apify push` command. The only thing `apify.json` does is sets the metadata for the push and build. Actors don't need an `apify.json` and if you're building code from GitHub, you don't need it at all. It's completely and utterly useless in that case.

Reply:
@mnmkng Ahh ok I thought it was required to build an actor on the platform. Will not focus on it